TYPO3.Neos/Documentation/GettingStarted/FeatureList.rst

```diff
@@ -7,20 +7,20 @@ Feature List
 
 .. note::
 
-	The following list contains the key features of Neos technical wise. If you want to learn about the great
-	features and experience that neos offers to editors and visitors please have look at the `Neos Website
+	The following list contains the key technical features of Neos. If you want to learn about the great
+	features and experience that neos offers to editors and visitors please have a look at the `Neos Website
 	<http://www.neos.io>`_.
 
 
 ContentRepository and ContentElements
 =====================================
 
-The content repository is conceptual core of Neos. It can store arbitrary content by managing so called Nodes that can
-have custom properties and childNodes. The list of available NodeTypes is extensible and hirarchical so new NodeTypes
+The content repository is the conceptual core of Neos. It can store arbitrary content by managing so called Nodes that can
+have custom properties and ChildNodes. The list of available NodeTypes is extensible and hierarchical so new NodeTypes
```

Comment:
child nodes are not a technical entity, merely a concept so should be "child nodes"